disnake/ext/commands/interaction_bot_base.py

```diff
@@ -421,6 +575,9 @@ def get_slash_command(
         ----------
         name: :class:`str`
             The name of the slash command to get.
+        guild_id: Optional[:class:`int`]
+            The guild ID corresponding to the slash command or ``None`` if it's a global command.
+            If this is not specified then the first match is returned instead.
```

Comment:
I personally feel that if multiple unique slash objects are found during the search then an error should be raised so the end user is aware of it instead of 'hoping' theres no duplicates returned in a search as I could see this resulting in weird end behaviour for some users otherwise

Reply:
Yeah this or a warning would make sense

Reply:
https://github.com/DisnakeDev/disnake/pull/1107/commits/e86e5cb646a2c3ba99c2bb0318acd3f2fec51572